server_src/runners/state.js

```diff
@@ -3459,12 +3458,13 @@ class Runner {
               serverRestartCommand: settings.systemCommands.serverRestart
             }
           },
+          // Due to now grabbing the updated state, the client won't have actually sent the "Other" settings tab so these would be undefined.
           webcam: {
-            webcamEnabled: settings.other.enableCamera,
-            timelapseEnabled: settings.other.enableTimeLapse,
-            rotate90: settings.other.rotateCamera,
-            flipH: settings.other.flipHCamera,
-            flipV: settings.other.flipVCamera
+            webcamEnabled: settings.other?.enableCamera,
+            timelapseEnabled: settings.other?.enableTimeLapse,
+            rotate90: settings.other?.rotateCamera,
+            flipH: settings.other?.flipHCamera,
+            flipV: settings.other?.flipVCamera
```

Comment:
These are here purely because currently when you send a save command with an offline printer the "other" object is never sent. Will clean up in the refactor.

Reply:
Sounds better to leave out the webcam section if `settings.other` is missing. The only reason this now doesnt change webcam is because `undefined` properties are removed in JS

Reply:
Yeah I agree, I will tackle that in the cleanup PR as currently what the client sends it a mess.